code/operators/operators.cpp

```diff
@@ -7,52 +7,62 @@ class Fraction {
 
 // TODO:operators
 
+
+void printAndCheck(const std::string & what, const Fraction & result, const Fraction & expected) {
+  const bool passed = result == expected;
+  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << result << "\n";
+}
+void printAndCheck(const std::string & what, bool result, bool expected) {
+  const bool passed = result == expected;
+  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << result << "\n";
+}
+
 int main() {
   // create a fraction with values 3 (which is 3/1) and 1/3
   const Fraction three{3};
   const Fraction athird{1, 3};
 
   // print the fractions
-  std::cout << "Three: " << three << '\n';
-  std::cout << "One third: " << athird << '\n';
+  printAndCheck("Three", three, Fraction{3, 1});
+  printAndCheck("One third", athird, Fraction{1, 3});
```

Comment:
Please leave the first two `cout` statements please, so students know better that this is the first operator to implement.